apps/omg/test/omg/state/transaction/recovered_test.exs

```diff
@@ -110,50 +112,50 @@ defmodule OMG.State.Transaction.RecoveredTest do
       assert {:error, :malformed_transaction} =
                Transaction.Recovered.recover_from(ExRLP.encode([sigs, inputs, outputs]))
 
-      assert {:error, :malformed_transaction} =
+      assert {:error, :unrecognized_transaction_type} =
                Transaction.Recovered.recover_from(ExRLP.encode([sigs, "bad_marker", inputs, outputs]))
 
       assert {:error, :malformed_witnesses} ==
-               Transaction.Recovered.recover_from(ExRLP.encode([[<<1>>, <<1>>], @payment_marker, inputs, outputs]))
+               Transaction.Recovered.recover_from(ExRLP.encode([[<<1>>, <<1>>], @payment_tx_type, inputs, outputs]))
```

Comment:
Same problem here with too many asserts, but you've only updated the values, not written the tests, so I guess it can be done in a future refactor/cleanup PR.